hack/test-update-storage-objects.sh

```diff
@@ -123,7 +123,7 @@ startApiServer ${KUBE_OLD_STORAGE_VERSIONS} ${KUBE_STORAGE_MEDIA_TYPE_JSON}
 
 
 # Create object(s)
-for test in ${tests[@]}; do
+for test in "${tests[@]}"; do
   IFS=',' read -ra test_data <<<"$test"
```

Comment:
Note IFS=,

Reply:
Thank you very much. @fejta 
You are right. I have updated the code.
:+1: